backend-docker/server/apiserver.go

```diff
@@ -123,8 +129,30 @@ func NewServer(cfg Config, docker dckr.Client) *Server {
 	return server
 }
 
+// remoteApiServer starts a server needed to access Docker remote API
```

Comment:
No, please use the docker client library that we already have, let's not reimplement the API.

````
func (c *Client) DownloadFromContainer(id string, opts DownloadFromContainerOptions) error
````

see https://github.com/fsouza/go-dockerclient/blob/master/container.go#L1200